blocks/library/gallery/editor.scss

```diff
@@ -1,4 +1,4 @@
-.wp-block-gallery:not( .components-placeholder ):not( .alignfull ) {
+.wp-block-gallery.wp-block-gallery:not( .components-placeholder ) {
```

Comment:
Seems the class is duplicated here?

Reply:
Hi @youknowriad, the class was duplicate to increase the specify against the styles set for the front end. I added a comment to make clear that this was intentional.

Reply:
In other blocks we use `.gutenberg .wp-block-myblock`, Can we adopt the same pattern? No strong opinion about which pattern to use but just trying to be consistent.